packages/cf-component-card/src/CardControl.js

```diff
@@ -3,16 +3,19 @@ const {PropTypes} = React;
 
 class CardControl extends React.Component {
   render() {
+    const className = 'cf-card__control' + (this.props.wide ? ' cf-card__control--wide' : '');
```

Comment:
I've found the [the `classnames` library](https://github.com/JedWatson/classnames) library to be useful at handling this particular oft-occurring problem.